src/app/modules/submission/encrypt-submission/__tests__/encrypt-submission.utils.spec.ts

```diff
@@ -101,4 +102,78 @@ describe('encrypt-submission.utils', () => {
       expect(newFilenames).toContain(`2-${validSingleFile.filename}`)
     })
   })
+
+  describe('getPaymentIntentDescription', () => {
+    it('should description for Fixed Payments Type', () => {
+      const expectedValue = 'expectedValue'
+      const formData = {
+        payments_field: {
+          payment_type: PaymentType.Fixed,
+          description: expectedValue,
+          name: 'name',
+        },
+      } as unknown as IPopulatedEncryptedForm
+
+      const products: any = [{}]
+
+      const result = getPaymentIntentDescription(formData, products)
+
+      expect(result).toEqual(expectedValue)
+    })
+    it('should return name for Variable Payment Type', () => {
+      const expectedValue = 'expectedValue'
+      const formData = {
+        payments_field: {
+          payment_type: PaymentType.Variable,
+          description: 'description',
+          name: expectedValue,
+        },
+      } as unknown as IPopulatedEncryptedForm
+
+      const products: any = [{}]
+
+      const result = getPaymentIntentDescription(formData, products)
+
+      expect(result).toEqual(expectedValue)
+    })
+
+    it('should return product names for Products Payment Type', () => {
+      const expectedValue = 'expectedValue'
+      const formData = {
+        payments_field: {
+          payment_type: PaymentType.Products,
+          description: 'description',
+          name: expectedValue,
+        },
+      } as unknown as IPopulatedEncryptedForm
+
+      const products: any = [
+        { data: { name: 'name1' }, quantity: 1 },
+        { data: { name: 'name2' }, quantity: 2 },
+      ]
+
+      const result = getPaymentIntentDescription(formData, products)
+
+      expect(result).toContain('name1')
```

Comment:
Maybe use a const for both the names instead of writing the same string twice?

Reply:
e6bc40a8a